types/sass-loader/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for sass-loader 8.0
+// Project: https://github.com/webpack-contrib/sass-loader
+// Definitions by: Jason Kwok <https://github.com/JasonHK>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.7
+
+import { LoaderOptions } from "./interfaces";
+
+declare function loader(content: string): void;
+
+declare namespace loader {
+    type Options = LoaderOptions;
+}
+
+export default loader;
```

Comment:
OK, so, while the `sass-loader` sources declare it as a `deafult`, the build plugin they're using actually make it the `module.export=` target of their whole module, which is obvious [at runtime.](https://npm.runkit.com/sass-loader)

To properly supper cjs consumers, use
```suggestion
export = loader;
```
instead. This does mean the locally, you'll need the `esModuleInterop` flag set to use it as a default.